bwresources.py

```diff
@@ -1196,7 +1196,7 @@ def upload_all(self, data_list, create_only=False, modify_only=False):
 
         for data in data_list:
             if "backfill" in data and data["backfill"] == True:
-                self.project.post(endpoint="rules/" + str(rules[data["name"]]) + "/backfill")
+                self.project.post(endpoint="bulkactions/rule/" + str(rules[data["name"]]))
```

Comment:
So the response is being discarded? That's OK then as it's different to the rule backfill response, but isn't used anyway.

Reply:
Yep - fire and forget